shell/app/modules/msp/monitor/status-insight/pages/status/status.tsx

```diff
@@ -37,10 +37,11 @@ const Status = () => {
 
   const { type = 'All' } = query || {};
 
-  const [{ modalVisible, formData, filterType }, updater, update] = useUpdate({
+  const [{ modalVisible, formData, filterType, mode }, updater, update] = useUpdate({
     modalVisible: false,
     formData: null as MONITOR_STATUS.IMetricsBody | null,
     filterType: type,
+    mode: '',
```

Comment:
we check whether its add/edit by whether formData is empty. no need add extra state

Reply:
ok,modified

Reply:
modified but state still there?

Reply:
mine,now modified